### ISSUE TITLE
SVG animation

### ISSUE THREAD

**Issue by JustinSainton** [wp-e-commerce/wpec-theme#7]
Lots of great usage of SVG in the theme, would be neat to animate some of that in a slick way: https://css-tricks.com/animating-svg-css/

**Comment by jptksc** [comment]
Yeah... for sure. Maybe I'll play a bit with SVG animation after we get all the other cleaned up.
